zcash_client_backend/src/data_api.rs

```diff
@@ -1317,10 +1317,10 @@ pub struct SentTransaction<'a, AccountId> {
     tx: &'a Transaction,
     created: time::OffsetDateTime,
     account: AccountId,
-    outputs: Vec<SentTransactionOutput<AccountId>>,
+    outputs: &'a Vec<SentTransactionOutput<AccountId>>,
```

Comment:
If you're taking them by reference, then use a slice instead of hard-coding `Vec` (to give the caller flexibility):
```suggestion
    outputs: &'a [SentTransactionOutput<AccountId>],
```